app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/FxAAccountOptionsView.java

```diff
@@ -91,6 +102,34 @@ protected void updateUI() {
         mBinding.footerLayout.setFooterButtonClickListener(v -> resetOptions());
     }
 
+    @Override
+    protected void onDismiss() {
+        if (!isEditing()) {
+            super.onDismiss();
+        }
+    }
+
+    @Override
+    public boolean isEditing() {
+        boolean editing = false;
+
+        if (mBinding.deviceNameEdit.isEditing()) {
+            editing = true;
+            mBinding.deviceNameEdit.cancel();
+        }
+
+        return editing;
+    }
+
+    @Override
+    public void onGlobalFocusChanged(View oldFocus, View newFocus) {
+        if (oldFocus != null) {
```

Comment:
Let's better early return and save indentation
```
if (oldFocus == null)
return;
```

Reply:
Actually, these are something that are copied from other optionsViews, maybe we want to change these things as well.